internal/scripts/fetch-crds

```diff
@@ -3,7 +3,7 @@
 PWD="$( cd $(dirname $0)/.. && pwd)"
 
 urls=(
-    "https://github.com/rabbitmq/cluster-operator/releases/download/v2.0.0/cluster-operator.yml"
+    "https://github.com/rabbitmq/cluster-operator/releases/latest/download/cluster-operator.yml"
```

Comment:
There are pros and cons to pinning versions. Can you share a bit about what made you move this to latest?

I am thinking about how we will manage clarity on what version this Promise is currently running with and how to make sure we are active about bumping this version regularly.

Reply:
Dependencies for this promise is included in the image. We won't accidentally bump the version unless we run this script, build and push a new version of the image, update the image version in promise.yaml.

Reply:
Agreed it won't move unexpectedly, but it can move without people knowing. And if we are looking to document the version, I would much rather see it documented in a repeatable build than just in the Readme as I worry we wouldn't remember to update it correctly.

This isn't a blocker per se, I just wanted to call out this change in philosophy as it can have knock on effects.